src/audio/coreaudio/SDL_coreaudio.m

```diff
@@ -769,21 +769,27 @@ static bool PrepareAudioQueue(SDL_AudioDevice *device)
             // L R C LFE Cs Ls Rs
             layout.mChannelLayoutTag = kAudioChannelLayoutTag_WAVE_6_1;
         } else {
-            // FIXME: We need to manually swizzle channels into a supported layout
             // L R C LFE Ls Rs Cs
-            //layout.mChannelLayoutTag = kAudioChannelLayoutTag_MPEG_6_1_A;
-            return SDL_SetError("Unsupported audio channels");
+            layout.mChannelLayoutTag = kAudioChannelLayoutTag_MPEG_6_1_A;
+
+            int swizzle_map[7] = {
```

Comment:
`static const` so it goes into the RODATA data section, and doesn't build on the stack.